products/mysql.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: MySQL
+title: Oracle MySQL
```

Comment:
Have never heard of this phrasing. -1

Reply:
i know it i am not the only one :D 
https://github.com/endoflife-date/endoflife.date/pull/2036#issuecomment-1350603622

Reply:
:trollface: